students/Sally_Shen/lesson2/gridPrinter.py

```diff
@@ -0,0 +1,39 @@
+
+'''Printing a Grid'''
+
+
+def grid(rowCols, dashes):
+    edge = '+'
+    row = '|'
+    for i in range(rowCols):
+        edge += '-' * dashes
+        edge += '+'
+        row += ' ' * dashes
+        row += '|'
+    edge += '\n'
+    row += '\n'
+
+    result = edge
+    for i in range(rowCols):
+        for j in range(dashes):
+            result += row
+        result += edge
+    print(result)
+
```

Comment:
they are a little tall and skinny -- the example has spaces between the dashes.

But you get the idea.

-CHB